docs/coding-style.md

```diff
@@ -98,6 +98,12 @@ Variable name `sut` (System Under Test) is recommended in unit tests that really
   * especially when each branch ends with a `return` statement.
 * Explicit `else` is not used after input validation.
 * Each compiler directive outside method body (namely `#if`/`#endif`) should be preceded and followed by an empty line.
+* When there are multiple conditions to be met before the core logic of a method can be executed, we have a preference in structuring these conditions:
+  * chained conditions in the same `if` statement together with positive logic ensure best readability (i.e. `if (condition) { doSomething(); }`) 
```

Comment:
The sentence defines chained conditions, but the example doesn't illustrate that. Let's do something like this.
Also `DoSomething()` instead of `doSometing()` to follow coding style 😄 
```suggestion
  * chained conditions in the same `if` statement together with positive logic ensure best readability (i.e. `if (first && second) { DoSomething(); }`) 
```